src/index.js

```diff
@@ -83,14 +90,14 @@ export function run (command, options = {}, logger = loggerAlias) {
   // Include in PATH node_modules bin path
   options.env.PATH = [binPath, options.env.PATH || process.env.PATH].join(path.delimiter)
 
-  logger.info(command)
```

Comment:
will do, I did it differently because of passing args and missed that on the cleanup.